src/ui/screens/ScreenGetHelp.qml

```diff
@@ -4,67 +4,66 @@
-        onMouseAreaClicked: function() {
-            if (unlockCounter >= 5) {
-                unlockCounter = 0
-                VPNSettings.developerUnlock = true
-            }
-            else if (!VPNSettings.developerUnlock) {
-                unlockTimeout.restart()
-                unlockCounter = unlockCounter + 1
-            }
-        }
-    }
+        spacing: 0
 
-    Timer {
-        id: unlockTimeout
-        repeat: false
-        running: false
-        interval: 10000
-        onTriggered: unlockCounter = 0
-    }
+        VPNMenu {
+            id: menu
+
+            property int unlockCounter: 0
 
-    VPNStackView {
-        id: getHelpStackView
+            objectName: "getHelpBack"
+            _menuOnBackClicked: () => VPNNavigator.requestPreviousScreen()
+            _menuIconButtonSource: getHelpStackView.depth === 1 ? "qrc:/nebula/resources/close-dark.svg" : "qrc:/nebula/resources/back.svg"
+            titleClicked: () => {
+                            if (unlockCounter >= 5) {
+                                unlockCounter = 0
+                                VPNSettings.developerUnlock = true
+                            }
+                            else if (!VPNSettings.developerUnlock) {
+                                unlockTimeout.restart()
+                                unlockCounter = unlockCounter + 1
+                            }
+                        }
 
-        anchors {
-            top: menu.buttom
+            Timer {
+                id: unlockTimeout
+                repeat: false
+                running: false
+                interval: 10000
+                onTriggered: menu.unlockCounter = 0
+            }
         }
 
-        implicitHeight: parent.height - menu.height
+        VPNStackView {
+            id: getHelpStackView
 
-        Component.onCompleted: function() {
-            VPNNavigator.addStackView(VPNNavigator.ScreenGetHelp, getHelpStackView)
-            getHelpStackView.push("qrc:/ui/screens/getHelp/ViewGetHelp.qml")
-        }
+            Layout.fillWidth: true
+            Layout.fillHeight: true
 
-        onCurrentItemChanged: {
-            menu.title = Qt.binding(() => currentItem._menuTitle || "");
-            menu.visible = Qt.binding(() => menu.title !== "");
-            menu._menuOnBackClicked = currentItem._menuOnBackClicked ? currentItem._menuOnBackClicked : () => getHelpStackView.pop()
+            Component.onCompleted: function() {
+                VPNNavigator.addStackView(VPNNavigator.ScreenGetHelp, getHelpStackView)
+                getHelpStackView.push("qrc:/ui/screens/getHelp/ViewGetHelp.qml")
+            }
+
+            onCurrentItemChanged: {
+                menu.title = Qt.binding(() => currentItem._menuTitle || "");
+                menu.visible = Qt.binding(() => menu.title !== "");
+                menu._menuOnBackClicked = currentItem._menuOnBackClicked ? currentItem._menuOnBackClicked : () => getHelpStackView.pop()
```

Comment:
Nit: Could this also be simplified:`menu._menuOnBackClicked || () => getHelpStackView.pop()`?